lib/rubocop/rspec/blank_line_separation.rb

```diff
@@ -8,6 +8,16 @@ module BlankLineSeparation
       include FinalEndLocation
       include RuboCop::Cop::RangeHelp
 
+      def check_missing_separating_lines(node, message:)
```

Comment:
Will it work with a default value `message: MSG`?

Reply:
No, that would refer to `BlankLineSeparation::MSG`.